README.md

```diff
@@ -7,916 +7,62 @@
-
-The tool running the Pebble server can make use of this, for example, under Kubernetes you could initialize its liveness and readiness probes to hit Pebble's `/v1/health` endpoint with `?level=alive` and `?level=ready` filters, respectively.
-
-If only a "ready" check or only an "alive" check is configured, ready implies alive, and not-alive implies not-ready. If you've configured an "alive" check but no "ready" check, and the "alive" check is unhealthy, `/v1/health?level=ready` will report unhealthy as well, and the Kubernetes readiness probe will act on that.
-
-On the other hand, not-ready does not imply not-alive: if you've configured a "ready" check but no "alive" check, and the "ready" check is unhealthy, `/v1/health?level=alive` will still report healthy.
-
-If there are no checks configured, the `/v1/health` endpoint returns HTTP 200 so the liveness and readiness probes are successful by default. To use this feature, you must explicitly create checks with `level: alive` or `level: ready` in the layer configuration.
-
-### Changes and tasks
-
-When Pebble performs a (potentially invasive or long-running) operation such as starting or stopping a service, it records a "change" object with one or more "tasks" in it. The daemon records this state in a JSON file on disk at `$PEBBLE/.pebble.state`.
-
-To see recent changes, for this or previous server runs, use `pebble changes`. You might see something like this:
-
-```
-$ pebble changes
-ID  Status  Spawn                Ready                Summary
-1   Done    today at 14:33 NZDT  today at 14:33 NZDT  Autostart service "srv1"
-2   Done    today at 15:26 NZDT  today at 15:26 NZDT  Start service "srv2"
-3   Done    today at 15:26 NZDT  today at 15:26 NZDT  Stop service "srv1" and 1 more
-```
-
-To drill down and see the tasks that make up a change, use `pebble tasks <change-id>`:
-
-```
-$ pebble tasks 3
-Status  Spawn                Ready                Summary
-Done    today at 15:26 NZDT  today at 15:26 NZDT  Stop service "srv1"
-Done    today at 15:26 NZDT  today at 15:26 NZDT  Stop service "srv2"
-```
-
-### Logs
-
-The daemon's service manager stores the most recent stdout and stderr from each service, using a 100KB ring buffer per service. Each log line is prefixed with an RFC-3339 timestamp and the `[service-name]` in square brackets.
-
-Logs are viewable via the logs API or using `pebble logs`, for example:
-
-```
-$ pebble logs
-2022-11-14T01:35:06.979Z [srv1] Log 0 from srv1
-2022-11-14T01:35:08.041Z [srv2] Log 0 from srv2
-2022-11-14T01:35:09.982Z [srv1] Log 1 from srv1
+""" > $PEBBLE/layers/001-http-server.yaml
+go run ./cmd/pebble run
```

Comment:
I agree it will be easier for the eyes. But I'm not sure if it's necessary, because the point here is that users can copy the whole block and run it. Thoughts? @benhoyt @tonyandrewmeyer

Reply:
Per our daily sync, we will add empty lines here.